# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1278008123 25200
#      Thu Jul 01 11:15:23 2010 -0700
# Node ID 5faf6832855a535b69f6bcd9311b204dea798740
# Parent  14af8004945d212a4964d75c2612381affae7ea8
refcard.tex: Fix italic macro typo for stem.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-07-01  Rik <octave@nomad.inbox5.com>
+
+        * refcard/refcard.tex: Fix italic macro typo for stem.
+
 2010-06-28  Rik <octave@nomad.inbox5.com>
 
         * refcard/refcard.tex: Fix typos in refcard (bug #30260).
 
 2010-06-16  Rik <octave@nomad.inbox5.com>
 
         * ChangeLog: Correctly break lines > 80 characters.
 
diff --git a/doc/refcard/refcard.tex b/doc/refcard/refcard.tex
--- a/doc/refcard/refcard.tex
+++ b/doc/refcard/refcard.tex
@@ -933,17 +933,17 @@ plot ({\it args})&2D plot with linear ax
 plot3 ({\it args})&3D plot with linear axes\cr
 line ({\it args})&2D or 3D line\cr
 patch ({\it args})&2D patch\cr
 semilogx ({\it args})&2D plot with logarithmic x-axis\cr
 semilogy ({\it args})&2D plot with logarithmic y-axis\cr
 loglog ({\it args})&2D plot with logarithmic axes\cr
 bar ({\it args})&plot bar charts\cr
 stairs ({\it x}, {\it y})&plot stairsteps\cr
-stem ({\it x}, {it y})&plot a stem graph\cr
+stem ({\it x}, {\it y})&plot a stem graph\cr
 hist ({\it y}, {\it x})&plot histograms\cr
 contour ({\it x}, {\it y}, {\it z})&contour plot\cr
 title ({\it string})&set plot title\cr
 axis ({\it limits})&set axis ranges\cr
 xlabel ({\it string})&set x-axis label\cr
 ylabel ({\it string})&set y-axis label\cr
 zlabel ({\it string})&set z-axis label\cr
 text ({\it x}, {\it y}, {\it str})&add text to a plot\cr
